RacingCar/Calculator.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Calculator.swift
+//  RacingCar
+//
+//  Created by main on 2022/05/02.
+//
+
+import Foundation
+
+enum CalculatorError: Error {
+    case divideByZero
+    case emptyString
+    case unknownOperator
+}
+
+class Calculator {
+    
+    private var evaluation: Int?
+    private var currentOperator: String?
+    
+    private func add(lhs: Int, rhs: Int) -> Int {
+        return lhs + rhs
+    }
+    
+    private func sub(lhs: Int, rhs: Int) -> Int {
+        return lhs - rhs
+    }
+    
+    private func mul(lhs: Int, rhs: Int) -> Int {
+        return lhs * rhs
+    }
```

Comment:
해당 부분이 명시된 [Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/#use-terminology-well) 문서 첨부해둡니다 👀

> Avoid abbreviations. Abbreviations, especially non-standard ones, are effectively terms-of-art, because understanding depends on correctly translating them into their non-abbreviated forms.